spec/models/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
We need to add specs for your instance methods:

`friends`
`send_friend_request`
`friend_requests`